data/mods/MindOverMatter/obsolete/powers.json

```diff
@@ -61,5 +104,49 @@
       ]
     },
     "ignored_monster_species": [ "PSI_NULL" ]
+  },
+  {
+    "id": "teleport_gateway_legacy",
+    "type": "SPELL",
+    "name": "[Ψ]Gateway (Obsolete)",
+    "description": "This power is Obsolete, but you can still use it.  Teleport to an attuned location.",
+    "message": "There is a long moment of darkness and terrible cold that seems like it will freeze your heart, and then you are somewhere else.",
+    "teachable": false,
+    "valid_targets": [ "self" ],
+    "spell_class": "TELEPORTER",
+    "skill": "metaphysics",
+    "flags": [ "PSIONIC", "CONCENTRATE", "SILENT", "NO_HANDS", "NO_LEGS" ],
+    "difficulty": 8,
+    "max_level": { "math": [ "int_to_level(1)" ] },
+    "effect": "translocate",
+    "shape": "blast",
+    "energy_source": "STAMINA",
+    "base_energy_cost": 10000,
+    "final_energy_cost": 6000,
+    "energy_increment": -200,
+    "base_casting_time": 200,
+    "final_casting_time": 75,
+    "casting_time_increment": -5.5
+  },
+  {
+    "id": "teleport_area_attune",
+    "type": "SPELL",
+    "name": "[Ψ]Attune to Area",
+    "description": "Attune to a location for later teleporting.",
+    "message": "",
+    "teachable": false,
+    "valid_targets": [ "ground" ],
+    "spell_class": "TELEPORTER",
+    "skill": "metaphysics",
+    "flags": [ "PSIONIC", "CONCENTRATE", "SILENT", "NO_HANDS", "NO_LEGS", "NO_FAIL" ],
+    "difficulty": 1,
+    "max_level": 1,
+    "effect": "ter_transform",
+    "effect_str": "ter_teleport_location_attune",
+    "shape": "blast",
+    "min_range": 1,
+    "energy_source": "STAMINA",
+    "base_energy_cost": 0,
+    "base_casting_time": { "math": [ "2880000 - max(( u_spell_level('teleport_gateway') * 108000), 180000)" ] }
```

Comment:
Unless the intention is that with a sufficiently high 'spell' level you can instantly attune, this formula needs some tweaks (the pre-PR formula has the same problem).

Reply:
This is not the first time I've used MAx when I meant to use Clamp, and I suspect it will not be the last.